frontend/hub/administration/remote-registries/RemoteRegistryForm.tsx

```diff
@@ -142,9 +143,16 @@ export function EditRemoteRegistry() {
     return (
       <PageLayout>
         <PageHeader
+          title={
+            RemoteRegistries?.name ? `${t('Edit')} ${RemoteRegistries?.name}` : t('Remote Registry')
```

Comment:
```suggestion
            RemoteRegistries?.name ? t('Edit {{registryName}}', {registryName: RemoteRegistries?.name}) : t('Remote Registry')
```

Embed the name in the t() string so translation can rearrange word order if needed. See useColumnAccess.tsx for an example